.scalafmt.conf

```diff
@@ -0,0 +1,2 @@
+version = "2.0.1"
```

Comment:
I'm fully onboard with using it, but if we remove it from here, it will be easier to rebase 0.4.0 on top of 0.3.1. And then we can include the formatting in 0.6.0.